### PR TITLE
Add docs about provide sulu twig attributes

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Fixed tickets | fixes -
| Related PRs | sulu/sulu#prnum
| License | MIT

#### What's in this PR?

Add docs about static controller routes.

#### Why?

Sometimes you have static routes which use the same base template

### CODE REVIEW

File: cookbook/static-route-controller.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+Create a static route controller
```
Comment (nnatter) [inline cookbook/static-route-controller.rst:1]: not sure if the term `static route controller` is used in the symfony docs. also you dont need to do this if you dont want to render your page template right?

Comment (nnatter) [reply cookbook/static-route-controller.rst:1]: maybe `Render page template from controller` / `Render base template from controller` or `Render page template from custom route`?

File: cookbook/static-route-controller.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+================================
+
+Sometimes you have static symfony routes like a custom search controller or any other
+route which is not a sulu content page.
```
Comment (nnatter) [inline cookbook/static-route-controller.rst:5]:
```suggestion
There are usecases where you want to render the template of your website outside of a sulu content page,
for example when rendering the results of a custom search controller.
```

File: cookbook/static-route-controller.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+That you can use the same ``base`` template for this static site you need to use the
+``TemplateAttributeResolver`` service which will generate you the required template attributes.
```
Comment (nnatter) [inline cookbook/static-route-controller.rst:8]:
```suggestion
In order to use the same ``base`` template for these custom routes, you need to provide the correct
attributes to your template. To do this, you can use the``TemplateAttributeResolver`` service in your controller:
```

File: cookbook/static-route-controller.rst (hunk 1 above)
Comment (alexander-schranz) [reply cookbook/static-route-controller.rst:1]: Should we go with something like "How to get sulu twig template attributes" `sulu-twig-attributes.rst`.

Comment (nnatter) [reply cookbook/static-route-controller.rst:1]: yes, thats good too 🙂
maybe `Provide Sulu twig template attributes in your controller`?

**Comment by alexander-schranz** [comment]
@nnatter docs updated

File: cookbook/sulu-twig-attributes.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
+Provide Sulu twig template attributes in your controller
+========================================================
+
+In order to use the same ``base`` template for these custom routes, you need to provide the correct
```
Comment (nnatter) [inline cookbook/sulu-twig-attributes.rst:4]:
```suggestion
In order to use the ``base`` template that is used for pages on custom routes, you need to provide the correct
```

**Pull request merged**